pkgs/development/python-modules/recordlinkage/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, buildPythonPackage, jellyfish, numpy, pandas, scipy, scikit-learn, joblib
+, networkx, bottleneck, numexpr, pyarrow, pytest, fetchPypi }:
+buildPythonPackage rec {
+  pname = "recordlinkage";
+  version = "0.14";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-kuY2MUuwaLb228kwkJmOnnU+OolZcvGlhKTTiama+T4=";
+  };
+
+  propagatedBuildInputs = [
+    pyarrow
+    jellyfish
+    numpy
+    pandas
+    scipy
+    scikit-learn
+    joblib
+    networkx
+    bottleneck
+    numexpr
+  ];
+
+  checkInputs = [ pytest ];
```

Comment:
I checked and it fails, reporting zero tests with no suggestive error message, alas.

Reply:
How are the tests started upstream?

Reply:
Like this: https://github.com/J535D165/recordlinkage/blob/master/.github/workflows/ci-workflow.yml#L36-L42

Reply:
Did you add the https://github.com/J535D165/recordlinkage/blob/master/.github/workflows/ci-workflow.yml#L41 to preCheck? If that does not help a little comment would be nice otherwise that question probably gets re-asked.

Reply:
@RaitoBezarius did you test this?

Reply:
@SuperSandro2000 Yes, I added the comment on the fact it does not work.